cypress/tests/ImageElement.spec.ts

```diff
@@ -60,6 +61,19 @@ describe("ImageElement", () => {
             .should("have.text", "deco");
         });
 
+        it(`${field} – should render new decorations, even if the document state has not changed`, () => {
+          addElement();
+          const newDecoString = "decoChanged";
+          const text = `${field} deco ${newDecoString}`;
+
+          typeIntoElementField(field, text);
+          changeTestDecoString(newDecoString);
+
+          getElementRichTextField(field)
```

Comment:
Test looks good, you could possibly improve it's robustness by setting the deco string back to old and asserting it changes back.